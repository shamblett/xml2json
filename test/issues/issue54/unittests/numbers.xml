<?xml version="1.0" encoding="utf-8"?>
<root>
  <zero>0</zero>
  <fixed>10.25</fixed>
  <octal>0123</octal>
  <hex>0x123</hex>
  <padded_zero>01</padded_zero>
  <fixed-neg>-10.25</fixed-neg>
  <int>123</int>
  <int-neg>-321</int-neg>
  <float>-12.123</float>
  <exp>-1.234e5</exp>
</root>
