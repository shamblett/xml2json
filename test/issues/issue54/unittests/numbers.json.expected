{"root":{"zero":0,"fixed":10.25,"octal":"0123","hex":"0x123","padded_zero":"01","fixed-neg":-10.25,"int":123,"int-neg":-321,"float":-12.123,"exp":-1.234e5}}
